app/src/main/java/io/mrarm/irc/IgnoreListAdapter.java

```diff
@@ -115,6 +125,17 @@ public void bind(ServerConfigData.IgnoreEntry entry) {
                 builder.append(" ");
                 builder.append(entry.comment, new ForegroundColorSpan(mTextColorSecondary));
             }
+
+            if (entry.mesg != null) {
+                if (builder.getSpannable().length() != 0) builder.append(" " );
```

Comment:
As I noted in the other pull request, this should be done in the XML using android:ellipsize="end" android:maxLines="1"